### PR TITLE
[release-3.0] Prepare release notes for 3.0.3

### PR DESCRIPTION
**Pull request by hardys**
Prepare release notes and related version bumps for the planned 3.0.3 release.

Note this is proposed directly to release-3.0 since main now only contains 3.1.x release information

### CODE REVIEW

**Comment by hardys** [comment]
Marking this as draft until the related artefacts are available, but review feedback is welcome, thanks!

File: asciidoc/guides/air-gapped-eib-deployments.adoc [hunk 1]
```diff
@@ -263,14 +263,14 @@ embeddedArtifactRegistry:
     - name: registry.rancher.com/rancher/hardened-etcd:v3.5.9-k3s1-build20240418
     - name: registry.rancher.com/rancher/hardened-flannel:v0.25.1-build20240423
     - name: registry.rancher.com/rancher/hardened-k8s-metrics-server:v0.7.1-build20240401
-    - name: registry.rancher.com/rancher/hardened-kubernetes:v1.28.10-rke2r1-build20240514
+    - name: registry.rancher.com/rancher/hardened-kubernetes:v1.28.13-rke2r1-build20240514
```
Comment (atanasdinov) [inline asciidoc/guides/air-gapped-eib-deployments.adoc:266]: Has someone tested this?

File: asciidoc/edge-book/releasenotes.adoc [hunk 2]
```diff
@@ -28,6 +28,100 @@ However, repeated entries are provided as a courtesy only. Therefore, if you are
 
 NOTE: SUSE Edge z-stream releases are tightly integrated and thoroughly tested as a versioned stack. Upgrade of any individual components to a different versions to those listed above is likely to result in system downtime. While it's possible to run Edge clusters in untested configurations, it is not recommended, and it may take longer to provide resolution through the support channels.
 
+= Release 3.0.2
```
Comment (atanasdinov) [inline asciidoc/edge-book/releasenotes.adoc:31]:
```suggestion
= Release 3.0.3
```

File: asciidoc/edge-book/version-matrix.adoc [hunk 3]
```diff
@@ -19,9 +19,9 @@ endif::[]
 | SLE Micro | 5.5 | N/A
 | Rancher Prime | 2.8.5 | 2.8.5
```
Comment (atanasdinov) [inline asciidoc/edge-book/version-matrix.adoc:20]: Not updated.

File: asciidoc/guides/air-gapped-eib-deployments.adoc (hunk 1 above)
Comment (atanasdinov) [reply asciidoc/guides/air-gapped-eib-deployments.adoc:266]: Nevermind. Rancher is supposed to be bumped too which means that this list of images has to be regenerated again. I'll do it later today, test it and DM you with the results.

File: asciidoc/edge-book/version-matrix.adoc [hunk 4]
```diff
@@ -19,9 +19,9 @@ endif::[]
...
 | Fleet | 0.9.2 | 0.9.2
```
Comment (ipetrov117) [inline asciidoc/edge-book/version-matrix.adoc:21]: IIUC Rancher `2.8.8` is running Fleet `0.9.9` their [build.yaml](https://github.com/rancher/rancher/blob/v2.8.8/build.yaml#L4)

File: asciidoc/guides/air-gapped-eib-deployments.adoc (hunk 1 above)
Comment (hardys) [reply asciidoc/guides/air-gapped-eib-deployments.adoc:266]: Thanks @atanasdinov - yes so far this is just a sed replacement, any help re-generating the list properly would be much appreciated!

File: asciidoc/edge-book/version-matrix.adoc (hunk 3 above)
Comment (hardys) [reply asciidoc/edge-book/version-matrix.adoc]: Done, thanks!

File: asciidoc/edge-book/version-matrix.adoc (hunk 4 above)
Comment (hardys) [reply asciidoc/edge-book/version-matrix.adoc]: Done, thanks!

File: asciidoc/edge-book/releasenotes.adoc [hunk 5]
```diff
@@ -28,6 +28,100 @@ However, repeated entries are provided as a courtesy only. Therefore, if you are
...
+* The K3s version is updated to `1.28.13`:  https://docs.k3s.io/release-notes/v1.28.X#release-v12813k3s1[Release Notes]
+* The Metal^3^ chart fixes an issue where with the handling of the `predictableNicNames` parameter: https://github.com/suse-edge/charts/pull/163[SUSE Edge issue #163]
+* The Metal^3^ chart resolves security issues identified in https://www.cve.org/CVERecord?id=CVE-2024-43803:[CVE-2024-43803]: https://github.com/suse-edge/charts/pull/163[SUSE Edge issue #163]
+* The Metal^3^ chart resolves security issues identified in https://www.cve.org/CVERecord?id=CVE-2024-44082:[CVE-2024-44082]: https://github.com/suse-edge/charts/pull/163[SUSE Edge issue #163]
+
+== Components Versions
+
+The following table describes the individual components that make up the 3.0.2 release, including the version, the Helm chart version (if applicable), and from where the released artifact can be pulled in the binary format. Please follow the associated documentation for usage and deployment examples. Note that items in bold are highlighted changes from the previous z-stream release.
```
Comment (fdegir) [inline asciidoc/edge-book/releasenotes.adoc:48]:
```suggestion
The following table describes the individual components that make up the 3.0.3 release, including the version, the Helm chart version (if applicable), and from where the released artifact can be pulled in the binary format. Please follow the associated documentation for usage and deployment examples. Note that items in bold are highlighted changes from the previous z-stream release.
```

**Review by fdegir — COMMENTED** [review]
Small issue with the release version, looks good otherwise.

File: asciidoc/guides/air-gapped-eib-deployments.adoc (hunk 1 above)
Comment (hardys) [reply asciidoc/guides/air-gapped-eib-deployments.adoc]: Updated with the list provided, many thanks!

**Review by ipetrov117 — DISMISSED** [review]

File: asciidoc/edge-book/version-matrix.adoc [hunk 6]
```diff
@@ -17,11 +17,11 @@ endif::[]
...
-| Rancher Prime | 2.8.5 | 2.8.5
-| Fleet | 0.9.2 | 0.9.2
-| K3s | 1.28.10 | N/A
-| RKE2 | 1.28.10 | N/A
-| Metal^3^ | 1.16.0 | 0.7.3
+| Rancher Prime | 2.8.8 | 2.8.8
+| Fleet | 0.9.9 | 0.9.9
+| K3s | 1.28.13 | N/A
+| RKE2 | 1.28.13 | N/A
+| Metal^3^ | 1.16.0 | 0.7.4
```
Comment (atanasdinov) [inline asciidoc/edge-book/version-matrix.adoc:24]: Is it a good time to get rid of the 1.16.0 version? It already caused misunderstandings with the marketing team when extracting the versions for the blog post.

File: asciidoc/guides/air-gapped-eib-deployments.adoc [hunk 7]
```diff
@@ -196,7 +196,7 @@ The `embeddedArtifactRegistry` section will include all images which are only re
 The <<components-rancher,Rancher>> deployment that will be demonstrated will be highly slimmed down for demonstration purposes. For your actual deployments, additional artifacts may be necessary depending on your configuration.
 ====
 
-The https://github.com/rancher/rancher/releases/tag/v2.8.5[Rancher v2.8.5] release assets contain a `rancher-images.txt` file which lists all the images required for an air-gapped installation.
+The https://github.com/rancher/rancher/releases/tag/v2.8.8[Rancher v2.8.8] release assets contain a `rancher-images.txt` file which lists all the images required for an air-gapped installation.
```
Comment (atanasdinov) [inline asciidoc/guides/air-gapped-eib-deployments.adoc:199]: This is no longer the case but let's merge this and I'll create a follow-up to address it.

**Review by atanasdinov — APPROVED** [review]

File: asciidoc/edge-book/version-matrix.adoc (hunk 6 above)
Comment (hardys) [reply asciidoc/edge-book/version-matrix.adoc:24]: Done

**Review by atanasdinov — APPROVED** [review]

**Pull request merged**